idr/connections.py

```diff
@@ -32,26 +32,15 @@ def connection(host=None, user=None, password=None, port=4064):
     c.createSession(user, password)
     conn = BlitzGateway(client_obj=c)
 
-    # You are now logged in. In order to guarantee
-    # that the connection is closed if a notebook
-    # does not complete, we are installing a handler
-    # below. This is ONLY for use in a notebook
-
-    # http://stackoverflow.com/questions/40110540/jupyter-magic-to-handle-notebook-exceptions
-    def custom_exc(shell, etype, evalue, tb, tb_offset=None):
-        print "Closing IDR connection..."
-        c.__del__()
-        shell.showtraceback((etype, evalue, tb), tb_offset=tb_offset)
     print "Connected to IDR..."
```

Comment:
Thinking ahead to this being a standalone library, we should try and avoid `print` statements since users might want to run this from a shell script and redirect/parse the output.

Reply:
Agreed. I've come to find these fairly annoying. That being said, I could see having each above the above as _additional_ functionality, but that not necessarily need impact this PR.